ns2-stat/src/lib.rs

```diff
@@ -20,17 +20,17 @@ impl<'a, I: Iterator<Item = &'a GameStats>> Iterator for Games<'a, I> {
 impl<'a, I: Iterator<Item = &'a GameStats>> Games<'a, I> {
     /// Filter the genuine games. This is done by ignoring games that took under 5 minutes
     /// and games that were likely bot games.
-    pub fn filter_genuine_games(self) -> Games<'a, impl Iterator<Item = &'a GameStats>> {
-        self.filter_on_length(|length| length >= 300.0).filter_bot_games()
+    pub fn genuine(self) -> Games<'a, impl Iterator<Item = &'a GameStats>> {
+        self.by_length(|length| length >= 300.0).bots()
     }
 
     /// Filter games with a predicate that takes the length of each game.
-    pub fn filter_on_length(self, f: impl Fn(f32) -> bool) -> Games<'a, impl Iterator<Item = &'a GameStats>> {
+    pub fn by_length(self, f: impl Fn(f32) -> bool) -> Games<'a, impl Iterator<Item = &'a GameStats>> {
```

Comment:
```suggestion
    pub fn filter_by_length(self, f: impl Fn(f32) -> bool) -> Games<'a, impl Iterator<Item = &'a GameStats>> {
```
I think `filter_by_name` is more appropriate, since this is merely a specialized `filter`.